Retail/src/retail/Retail.java

```diff
@@ -0,0 +1,107 @@
+package retail;
+import java.util.Scanner;
+import retailPackage.RetailItem;
+
+public class Retail {
+
+    /**
+     * Main program that calls the retail class
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        Scanner keyboard = new Scanner(System.in);
+        
+        //variable init
+        String description;
+        int onHand;
+        double price;
+        
+        print("Please enter the First Items Description: ");
+        description = keyboard.nextLine();//description input
```

Comment:
Continuous reassignment of these variables is a little risky. There is a principle called DRY (don't repeat yourself). These sections are a prime example of where some abstraction can clean up these repeated sections and assure that the logic is contained and reduce side effects.